sf-fx-runtime-java-logger/src/main/java/com/salesforce/functions/jvm/runtime/logger/DefaultLoggingFormatter.java

```diff
@@ -24,16 +26,24 @@ public String format(String loggerName, Level level, String message) {
     String invocationId = MDC.get("function-invocation-id");
     String localDateTimeString = LocalDateTime.now(clock).format(DATE_TIME_FORMATTER);
 
-    return String.format(
-        FORMAT_STRING,
-        localDateTimeString,
-        level.toString(),
-        invocationId,
-        Utils.shortenLoggerName(loggerName, 36),
-        message);
+    String formatString =
+        String.format(
+            FORMAT_STRING,
+            localDateTimeString,
+            level.toString(),
+            invocationId,
+            Utils.shortenLoggerName(loggerName, 36),
+            message);
+    Map<String, char[]> parsedString = Logfmt.parse(formatString.toCharArray());
+    StringBuilder formattedString = new StringBuilder();
+    for (String value : parsedString.keySet()) {
+      formattedString.append(value).append("=").append(parsedString.get(value)).append(" ");
+    }
+    return formattedString.toString();
   }
 
-  private static final String FORMAT_STRING = "%s %-5s [INVOCATION %s] %s - %s\n";
+  private static final String FORMAT_STRING =
+      "localDateTime=%s level=%s invocationId=%s loggerName=%s message=\"%s\"\n";
```

Comment:
Not an issue introduced in this PR, but something we need to fix regardless (also in Node.js): using a local date time string will make it harder to debug issues. The language runtimes are used in a highly distributed solution. I suggest we move this to UTC (ideally ISO 8601) and change the `logfmt` key to something that reflects this.

Reply:
@Malax should we convert it before we store it the map which logfmt parses into?